packages/components/atoms/f-error-boundary/src/components/tests/ErrorBoundary.test.js

```diff
@@ -0,0 +1,140 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import ErrorBoundary from '../ErrorBoundary.vue';
+
+const localVue = createLocalVue();
+
+const createComponent = ({
+    propsData,
+    computed,
+    scopedSlots
+} = {}) => shallowMount(ErrorBoundary, {
+    localVue,
+    computed,
+    propsData,
+    scopedSlots
+});
+
+describe('ErrorBoundary', () => {
+    const error = 'test error';
+    const vm = 'test vm';
+    const info = 'test info';
+    const tier = 'test tier';
+    const loggerPayload = 'test payload';
+
+    describe('render', () => {
+        const scopedSlots = {
+            default: `<div>
+                <p>hasError: {{ props.hasError }}</p>
+                <p>error: {{ props.error }}</p>
+                <p>vm: {{ props.vm }}</p>
+                <p>info: {{ props.info }}</p>
+                <p>tier: {{ props.tier }}</p>
+                <p>loggerPayload: {{ props.loggerPayload }}</p>
+            </div>`
+        };
+
+        describe('when `hideOnError` is false and `hasError` is false', () => {
```

Comment:
Worth expanding this to be an `.each()` over `false`&`false`/`true`&`false`/`false`&`true` condition sets?

Reply:
Done 👍🏻